parser-typechecker/src/Unison/Parser.hs

```diff
@@ -33,8 +33,31 @@ import qualified Unison.Var as Var
 debug :: Bool
 debug = False
 
-type PEnv = UnisonFile.CtorLookup
-type P v = P.ParsecT (Error v) Input ((->) PEnv)
+data PEnv v = PEnv { constructorLookup :: UnisonFile.CtorLookup }
```

Comment:
Phantom type v?  How is it used, or is its use coming later?

Reply:
Maybe it's just for the Monoid / Semigroup?  I did the same thing for `Rendered`.